diff --git a/node_modules/meriyah/dist/src/meriyah.d.ts b/node_modules/meriyah/dist/src/meriyah.d.ts
index 5ce9c5b..55c3b7b 100644
--- a/node_modules/meriyah/dist/src/meriyah.d.ts
+++ b/node_modules/meriyah/dist/src/meriyah.d.ts
@@ -1,6 +1,6 @@
 import { Options } from './parser';
 import * as ESTree from './estree';
-import { version } from '../package.json';
+import { version } from '../../package.json';
 export declare function parseScript(source: string, options?: Options): ESTree.Program;
 export declare function parseModule(source: string, options?: Options): ESTree.Program;
 export declare function parse(source: string, options?: Options): ESTree.Program;
diff --git a/node_modules/meriyah/dist/src/parser.d.ts b/node_modules/meriyah/dist/src/parser.d.ts
index bd59988..2fdc16a 100644
--- a/node_modules/meriyah/dist/src/parser.d.ts
+++ b/node_modules/meriyah/dist/src/parser.d.ts
@@ -15,7 +15,7 @@ export interface Options {
     lexical?: boolean;
     source?: string;
     jsx?: boolean;
-    onComment?: Comment[] | OnComment;
+    onComment?: ESTree.Comment[] | OnComment;
     onInsertedSemicolon?: OnInsertedSemicolon;
     onToken?: Token[] | OnToken;
     uniqueKeyInPattern?: boolean;
